enterprise/app/settings/settings.css

```diff
@@ -145,10 +145,38 @@
   border-bottom: 1px solid #eee;
 }
 
+.settings .github-app-installation .installation-header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+}
+
 .settings .github-app-installation .installation-owner {
   margin-right: auto;
 }
 
+.settings .github-app-installation .installation-settings {
+  display: flex;
+  flex-direction: column;
+  margin-left: 16px;
```

Comment:
Delete this extra margin so that the checkbox lines up with the repo name? Elsewhere in the UI, we don't use whitespace/indentation like this to indicate hierarchy.

![image](https://github.com/user-attachments/assets/fcb763a1-f005-4b61-a627-3d79f7a5bc58)


```suggestion
```

Reply:
Ack - updated the image in the description